tests/publiccloud/upload_image.pm

```diff
@@ -32,6 +32,12 @@ sub run {
         return;
     }
 
+    # Check if remote file exists
+    my $res = script_output("wget -q --spider -S $img_url");
+    if ($res =~ m@HTTP/1.1 404 Not Found@) {
+        die "Image was not found";
```

Comment:
```suggestion
        die "Image was not found in IBS";
```

Reply:
I find IBS misleading, as for OpenStack we don't pull it from there by the test. Maybe I can print also the content of `$img_url` in the message